bookmarks/migrations/0001_initial.py

```diff
@@ -10,23 +10,23 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
+        ('grades', '0002_auto_20201219_2207'),
```

Comment:
This is dependent on a file thqat isn't in the repo or PR. I suspect it won't work

Reply:
Those dependencies were automatically created after running makemigrations. Should I just delete '0002_auto_20201219_2207'?

Reply:
probably. It auto created it probably because you had a file in the migrations directory with that name

Reply:
I tried to do that but the migrations won't work without it.

Reply:
migrations can get complicated. For sure it won't work if 0002_auto_20201219_2207 isn't in your repo :-)
Sometimes you need to erase the database, make sure the folder only has the files you want and then call makemigrations.

Reply:
@Noa-Git you have to choose:
0001_initial created in makmigrate from model.py

Check if in setup.sh you have makemigrate,
You can delete 0001_initial,py
Or if you use makemigrate in script, remove 0001_initial.py it will be created automatically
You can call 0001_initial.py from the the test

Reply:
@Yarboa 
In setup.sh I have migrate and not makemigrations because migrate itself is updating the db and makemigrations is used just for the creation of the database migrations right?
As for removing '0002_auto_20201219_2207' file- I deleted the migrations files, the db.sqlite3 and then run makemigrations and migrate but nothing has changed

Reply:
So it seems that some one add _auto_ and initial, during the local makemigrate
I would advise, remove all migration dir, run makemaigrate,
You will have new 0001_initial.py
Add the 0002_test run maigrate.


If you have no makemigrate, add both files under migrations dirs to your patch,
Note, do not forget to rm db.sqlite3 before you do that
Good luck

Reply:
@Yarboa you were right. some one did add _auto_ during the local makemigrate. I tried what you have suggested but it won't work, so I'll fix the dependencies once she'll upload a PR.

Reply:
Can you explain not work? you shoud have 0001_initial.py after make migrate
Add 0002_test and run migrate
In case of assistance needed, please ping me on slack

Reply:
> I'm waiting for Meital's PR to approved since we found out that under migrations files there was 'grades.0002_auto_20201219_2207'. She deleted it locally and it's a part of her PR now. Been trying to follow Yariv's instructions but the file is still there.
> @simzacks

It is useful to add the PR link which this PR is depended on.

Reply:
#62

Reply:
Problem solved at last (without Meital's PR merged )

Reply:
The file is now in the repo

Reply:
I'll delete all unnecessary migration files that were added by mistake in a different PR